app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala

```diff
@@ -67,7 +67,7 @@ class MappingController @Inject()(override val messagesApi: MessagesApi,
         successful(Ok(html.add_code(formWithErrors, request.saAgentReference)))
       },
       mappingData => {
-        mappingConnector.createMapping(mappingData.arn, request.saAgentReference) map {_ =>
+        mappingConnector.createMapping(Utr(mappingData.utr), mappingData.arn, request.saAgentReference) map {_ =>
           Redirect(routes.MappingController.complete())
```

Comment:
Should we change MappingForm so it has utr as a typed Utr not a String to be consistent with Arn?